requirements.txt

```diff
@@ -2,7 +2,7 @@ future
 zsl_client
 click>=6.6
 Flask>=0.11
-Flask-Injector==0.8
+Flask-Injector>=0.8
```

Comment:
Thank you, I wanted to inspect the issue. However, we could make a switch - Py 2.7 = injector ==0.8 and Py 3 gives the latest version. Wouldn't it be possible?